### PR TITLE
fix: [CI-3672]: Enabled runtime input support for `Report Paths` and `Output Variables` fields

### PR DESCRIPTION
**Pull request by mtran7**
##### Summary:

Enabled runtime input support for `Report Paths` and `Output Variables` fields.

##### Jira Links:

https://harness.atlassian.net/browse/CI-3672

##### Screenshots:

image

image

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by mtran7** [comment]
retrigger titlecheck

**Comment by mtran7** [comment]
trigger cypress

**Review by vardanbansal-harness — APPROVED** [review]

File: src/modules/75-ci/components/PipelineSteps/CIStep/CIStepOptionalConfig.tsx [hunk 1]
```diff
@@ -318,15 +370,17 @@ export const CIStepOptionalConfig: React.FC<CIStepOptionalConfigProps> = props =
...
-        <Container className={cx(css.formGroup, stepCss, css.bottomMargin5)}>
-          {renderMultiTypeList({
-            name: `${prefix}spec.outputVariables`,
-            labelKey: 'pipelineSteps.outputVariablesLabel',
-            allowedTypes: [MultiTypeInputType.FIXED]
-          })}
-        </Container>
-      ) : null}
+      {/* Input Set render for Report Paths is handled in respective steps itself and not handled here due to a technical limitation */}
+      {Object.prototype.hasOwnProperty.call(enableFields, 'spec.outputVariables') &&
```
Comment (vardanbansal-harness) [inline src/modules/75-ci/components/PipelineSteps/CIStep/CIStepOptionalConfig.tsx:374]: `Report Paths` -> `Output Variables` this typo will be fixed in next PR.

Comment (vardanbansal-harness) [inline src/modules/75-ci/components/PipelineSteps/CIStep/CIStepOptionalConfig.tsx:374]: `Report Paths` -> `Output Variables` this typo will be fixed in next PR.

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**